test/game.test.js

```diff
@@ -0,0 +1,69 @@
+const Game = require('../lib/game')
+
+describe('Game class', () => {
+  beforeEach(() => {
+    game = new Game()
+  })
+
+  it('create an instance of Game', () => {
+    expect(game).toBeInstanceOf(Game)
+  })
+
+  it('initialize with an empty array', () => {
+    expect(game.frames).toEqual([])
+  })
+
+  describe('addRolls', () => {
+    it('add the rolls to the frames array', () => {
+      game.addRolls(1, 4)
+      expect(game.frames).toEqual([[1, 4]])
+    })
+
+    it('does not add the rolls and throws error Sum of the rolls cannot exceed 10', () => {
+      expect(() => game.addRolls(5,6)).toThrow('Sum of the rolls cannot exceed 10')
+      expect(game.frames).toEqual([])
+    })
```

Comment:
I'm guessing this would also work  (i.e. throw error) if I put my first bowl as 11?